packages/core/src/new-api/types/module-builder.ts

```diff
@@ -42,7 +44,7 @@ export interface ContractOptions {
   after?: Future[];
   libraries?: Record<string, ContractFuture<string>>;
   value?: bigint;
-  from?: string;
+  from?: string | AccountRuntimeValue;
```

Comment:
`from` can accept an account obtained with `m.getAccount()` or `string`. In theory it could accept a `Future` or module param, but that doesn't seem like a realistic use-case.